src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -121,9 +121,10 @@
  * ///////////////////////////////////////////////////
  * package com.whitespace; // violation, 'package' should be separated from previous line.
  * import java.io.Serializable; // violation, 'import' should be separated from previous line.
- * class Foo { // violation, 'CLASS_DEF' should be separated from previous line.
- *   public static final int FOO_CONST = 1;
- *   public void foo() {} // violation, 'METHOD_DEF' should be separated from previous line.
+ * class FirstClass { // violation, 'CLASS_DEF' should be separated from previous line.
+ *   public static final int firstClassVariable = 1;
```

Comment:
ok, as we did big rafacotring.

please make this line to be something like:
`public int field;`
to be short and let user focus on what is matter.

Reply:
Done. Respective changes have been made at all the places of concern.